code/game/gamemodes/clock_cult/clock_effects/city_of_cogs_rift.dm

```diff
@@ -46,7 +46,13 @@
 
 /obj/effect/clockwork/city_of_cogs_rift/proc/beckon(atom/movable/AM)
 	var/turf/T = get_turf(pick(GLOB.city_of_cogs_spawns))
-	if(is_servant_of_ratvar(AM))
+	var/servant_pulled = AM.pulledby && is_servant_of_ratvar(AM.pulledby)
+	if(AM.fingerprintslast)
+		for(var/V in GLOB.player_list)
```

Comment:
why not loop the list of servants?

Reply:
Because I wrote this code really tired and it never occurred to me.

Reply:
always the simple reasons